traffic_ops/traffic_ops_golang/topology/topologies.go

```diff
@@ -781,13 +785,13 @@ WHERE tcp.child IN
 func updateQuery() string {
 	query := `
 UPDATE topology t SET
-description = $1
-WHERE t.name = $2
+name = $1,
+description = $2
+WHERE t.name = $3
 RETURNING t.name, t.description, t.last_updated
 `
 	return query
 }
-
```

Comment:
gofmt will probably complain about this line missing

Reply:
oh ok.. i'll add it.